frontend/src/components/TimeLogsPage/TimeLogsPage.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { TimeLogForm } from './TimeLogForm'
+import { TimeLogRow } from './TimeLogRow'
+import {
+  NotInGroupPlaceholder,
+  LoadingPlaceholder,
+} from '../common/Placeholders'
+import { SprintSelect } from './SprintSelect'
+import { Typography } from '@material-ui/core'
+import timeLogsService from '../../services/timeLogs'
+import {
+  minutesAndHoursFromString,
+  hoursAndMinutesToMinutes,
+} from '../../utils/functions'
+
+import './TimeLogsPage.css'
+
+const TimeLogsPage = (props) => {
+  const [allLogs, setAllLogs] = useState([])
+  const [sprintNumber, setSprintNumber] = useState(1) // TODO: determine current sprint.
```

Comment:
Let's do this in the Sprint Management PR

Reply:
Yes, but for now should be left as it is, right?

Reply:
Yes I think so!